frontend/src/reports/ui/NoDataAlert.tsx

```diff
@@ -23,26 +24,23 @@ function NoDataAlert(props: { dropdownVarId: string }) {
         >
           See our roadmap to learn more.
         </Button>
-      </Alert>
-      <Alert variant='outlined' severity='info'>
+      </HetNotice>
+      <HetNotice>
         Do you have information on {props.dropdownVarId} at the state or local
         level?
-        <Button
+        <a
           style={{
             padding: '0',
             paddingLeft: '5px',
             paddingRight: '5px',
             background: 'none',
             textDecoration: 'underline',
           }}
```

Comment:
Were you able to convert this to tailwind?

Reply:
i was going to leave that for a follow up PR; trying to keep this one scoped to just the Alert switchovers even though i failed and definitely added some other refactors in lol.